test/gtest/deepbench_lstm.cpp

```diff
@@ -108,40 +106,67 @@ TEST_P(ConfigWithFloat, FloatTest)
-    {flags + " --batch-size 32 --seq-len 150 --vector-len 256 --hidden-size 256" + commonFlags},
-    {flags + " --batch-size 64 --seq-len 150 --vector-len 256 --hidden-size 256" + commonFlags}
-        // clang-format on
-    };
+        {flags + " --batch-size 16 --seq-len 25 --vector-len 512 --hidden-size 512" + commonFlags},
+        {flags + " --batch-size 32 --seq-len 25 --vector-len 512 --hidden-size 512" + commonFlags},
+        {flags + " --batch-size 64 --seq-len 25 --vector-len 512 --hidden-size 512" + commonFlags},
+        {flags + " --batch-size 128 --seq-len 25 --vector-len 512 --hidden-size 512" + commonFlags},
+        {flags + " --batch-size 16 --seq-len 25 --vector-len 1024 --hidden-size 1024" +
+         commonFlags},
+        {flags + " --batch-size 32 --seq-len 25 --vector-len 1024 --hidden-size 1024" +
+         commonFlags},
+        {flags + " --batch-size 64 --seq-len 25 --vector-len 1024 --hidden-size 1024" +
+         commonFlags},
+        {flags + " --batch-size 128 --seq-len 25 --vector-len 1024 --hidden-size 1024" +
+         commonFlags},
+        {flags + " --batch-size 16 --seq-len 25 --vector-len 2048 --hidden-size 2048" +
+         commonFlags},
+        {flags + " --batch-size 32 --seq-len 25 --vector-len 2048 --hidden-size 2048" +
+         commonFlags},
+        {flags + " --batch-size 64 --seq-len 25 --vector-len 2048 --hidden-size 2048" +
+         commonFlags},
+        {flags + " --batch-size 128 --seq-len 25 --vector-len 2048 --hidden-size 2048" +
+         commonFlags},
+        {flags + " --batch-size 16 --seq-len 25 --vector-len 4096 --hidden-size 4096" +
+         commonFlags},
+        {flags + " --batch-size 32 --seq-len 25 --vector-len 4096 --hidden-size 4096" +
+         commonFlags},
+        {flags + " --batch-size 64 --seq-len 25 --vector-len 4096 --hidden-size 4096" +
+         commonFlags},
+        {flags + " --batch-size 128 --seq-len 25 --vector-len 4096 --hidden-size 4096" +
+         commonFlags},
+        {flags + " --batch-size 8 --seq-len 50 --vector-len 1536 --hidden-size 1536" + commonFlags},
+        {flags + " --batch-size 16 --seq-len 50 --vector-len 1536 --hidden-size 1536" +
+         commonFlags},
+        {flags + " --batch-size 32 --seq-len 50 --vector-len 1536 --hidden-size 1536" +
+         commonFlags},
+        {flags + " --batch-size 16 --seq-len 150 --vector-len 256 --hidden-size 256" + commonFlags},
+        {flags + " --batch-size 32 --seq-len 150 --vector-len 256 --hidden-size 256" + commonFlags},
+        {flags + " --batch-size 64 --seq-len 150 --vector-len 256 --hidden-size 256" +
+         commonFlags}};
 
     return test_cases;
 }
 
-INSTANTIATE_TEST_SUITE_P(ConvTrans, ConfigWithFloat, testing::Values(GetTestCases()));
+INSTANTIATE_TEST_SUITE_P(DeepbenchLstm, ConfigWithFloat, testing::Values(GetTestCases("--float")));
+
+INSTANTIATE_TEST_SUITE_P(DeepbenchLstm, ConfigWithHalf, testing::Values(GetTestCases("--half")));
```

Comment:
[Notice] HALF is not required for deepbench but I am not against this.

/cc @shurale-nkn

Reply:
[resolved]